### PR TITLE
[TEST] Add tensor common unittests

### PR DESCRIPTION
**Pull request by gichan-jang**
Add unittests for tensor common functions.

Signed-off-by: Gichan Jang <gichan2.jang@samsung.com>

Self evaluation:
1. Build test: [ *]Passed [ ]Failed [ ]Skipped
2. Run test: [ *]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: tests/common/unittest_common.cc [hunk 1]
```diff
@@ -577,6 +577,202 @@ TEST (commonTensorInfo, equal05_n)
...
+  EXPECT_FALSE (gst_tensor_info_is_equal (NULL, &info));
+}
+
+/**
+ * @brief Test for comparing two tensor info with invalid param.
+ */
+TEST (commonTensorInfo, equalInvalidParam1_n)
+{
+  GstTensorInfo info;
+  EXPECT_FALSE (gst_tensor_info_is_equal ( &info, NULL));
```
Comment (dongju-chae) [inline tests/common/unittest_common.cc:619]: Unnecessary space detected

File: tests/common/unittest_common.cc [hunk 2]
```diff
@@ -577,6 +577,202 @@ TEST (commonTensorInfo, equal05_n)
...
+  EXPECT_FALSE (gst_tensors_info_is_equal (NULL, &info));
+}
+
+/**
+ * @brief Test for comparing two tensors info with invalid param.
+ */
+TEST (commonTensorsInfo, equalInvalidParam1_n)
+{
+  GstTensorsInfo info;
+  EXPECT_FALSE (gst_tensors_info_is_equal ( &info, NULL));
```
Comment (dongju-chae) [inline tests/common/unittest_common.cc:637]: ditto

File: tests/common/unittest_common.cc (hunk 1 above)
Comment (gichan-jang) [reply tests/common/unittest_common.cc]: Updated!

**Review by dongju-chae — APPROVED** [review]
LGTM

**Review by jaeyun-jung — APPROVED** [review]

**Pull request merged**